lms/envs/common.py

```diff
@@ -443,6 +443,19 @@
     # .. toggle_warnings: Also set settings.ACCOUNT_MICROFRONTEND_URL and set REDIRECT_TO_ACCOUNT_MICROFRONTEND for rollout.
     'ENABLE_LOGISTRATION_MICROFRONTEND': False,
 
+    # .. toggle_name: ENABLE_THIRD_PARTY_ONLY_AUTH
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to prevent using username/password login and registration and only allow authentication with third party auth
+    # .. toggle_category: admin
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2020-05-13
+    # .. toggle_expiration_date: None
+    # .. toggle_tickets: None
+    # .. toggle_status: supported
+    # .. toggle_warnings: Requires configuration of third party auth
+    'ENABLE_THIRD_PARTY_ONLY_AUTH': False,
```

Comment:
Can we rename this toggle to `ENABLE_REQUIRE_THIRD_PARTY_AUTH`?

Rationale: "3rd party auth" is generally an industry known phrase. So keeping that phrase intact makes it easier to read this identifier.